package/Dockerfile.nettest

```diff
@@ -2,6 +2,17 @@ FROM alpine
 
 WORKDIR /app
 
+RUN apk add --update --no-cache gcc libc-dev make curl
+
+RUN curl -L https://github.com/HewlettPackard/netperf/archive/netperf-2.7.0.tar.gz | tar xzf -
+RUN cd netperf-netperf-2.7.0 && ./configure \
+    && make && make install
+
```

Comment:
```suggestion
```

OCD

Reply:
That was done on purpose, to separate the two images.